src/pricing-fetcher.ts

```
@@ -188,6 +188,29 @@ export class PricingFetcher implements Disposable {
 		);
 	}
 
+	/**
+	 * Gets context window limit for a specific model from LiteLLM data
+	 * @param modelName - The model name to get context limit for
+	 * @returns The context limit in tokens, or null if not found
+	 */
+	async getModelContextLimit(modelName: string): Result.ResultAsync<number | null, Error> {
+		return Result.pipe(
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)